react/src/components/atoms/forms/InputCurrency/index.js

```diff
@@ -75,7 +72,7 @@ const Currency = (props) => {
               update.showError = false;
               update.errorMsg = errorMsg;
             }
-            context.updateState(update, () => {
+            context.updateOwnState(update, () => {
               if (typeof props.onChange === 'function') {
```

Comment:
can we update this to take advantage of new is dependency?
```suggestion
              if (props.onChange && is.fn(props.onChange)) {
```